### ISSUE TITLE
Register new digital identity

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#6]
## User story
1. As a user (HR employee)
2. I want / need to register an employee 
3. So that I can enter his personal information

## Acceptance criteria
* Have a tab for registering employees
* 
* 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
